Add support for html_show_copyright and html_show_sphinx

By default the guzzle them hard codes the 'copyright' and 'built by
sphinx' text in the footer.  In regular sphinx this text is controlled
by the boolean configuration options 'html_show_copyright and
'html_show_sphinx'.

This patch restores these options to the theme.

diff --git a/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html b/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html
index 85be1c5..f35a892 100644
--- a/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html
+++ b/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html
@@ -153,7 +153,16 @@
 <script type="text/javascript" src="{{ pathto('_static/js/bootstrap.js', 1)}}"></script>
 {%- block footer_wrapper %}
   <div class="footer">
-    &copy; Copyright {{ copyright }}. Created using <a href="http://sphinx.pocoo.org/">Sphinx</a>.
+    {%- if show_copyright %}
+      {%- if hasdoc('copyright') %}
+        {% trans path=pathto('copyright'), copyright=copyright|e %}&copy; <a href="{{ path }}">Copyright</a> {{ copyright }}.{% endtrans %}
+      {%- else %}
+        &copy; Copyright {{ copyright }}.
+      {%- endif %}
+    {%- endif %}
+    {%- if show_sphinx %}
+    Created using <a href="https://github.com/guzzle/guzzle_sphinx_theme/">Guzzle Sphinx Theme</a>.
+    {%- endif %}
   </div>
 {%- endblock %}
 {%- block ga %}
